server/migrations/1653614479846_add-post-hire-is-current.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable camelcase */
+
+const { collections } = require('../db');
+
+exports.shorthands = 'add_is_current_post_hire_status';
+
+exports.up = (pgm) => {
+  pgm.addColumns(
+    collections.PARTICIPANT_POST_HIRE_STATUS,
+    {
+      is_current: {
+        type: 'boolean',
+        notNull: true,
+        default: true,
```

Comment:
1. Default should be false 
2. You need to create another migration to set true for the latest status.
Why?
Because in our prod DB there can be more than one post_hire_statuses for a participant